lib/02_calculator.rb

```diff
@@ -0,0 +1,41 @@
+def add(a,b)
```

Comment:
Style comment - usually there is a space between arguments like the ones passed in here. 

`a, b`